site/content/Qasidas/Hibbi-Injala-Jamaluh.md

```diff
@@ -1,5 +1,10 @@
++++
+maqam = ['Kurd']
```

Comment:
fyi @thenotepadwriter we can do this now, probably the meta information should be moved to the 'About' section